Build.ps1

```diff
@@ -23,7 +23,7 @@ foreach ($src in ls src/*) {
 
     echo "build: Packaging project in $src"
     
-    foreach ($tfm in @("netstandard2.0", "net6.0")) {
+    foreach ($tfm in @("net6.0", "netstandard2.0")) {
```

Comment:
Couldn't this change be removed?

Reply:
This change gives the package a chance of working on pre-2022.1 Seq versions; it's probably easier to leave this in place for now, and adjust in the future if we want to narrow down/tighten up Seq version support.